src/client/components/app/Transactions/components/TxError.tsx

```diff
@@ -1,62 +1,151 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 import styled from 'styled-components';
 
-import { Icon, Text, WarningIcon } from '@components/common';
+import { ChevronTxIcon, ErrorIcon, Icon, Text, WarningIcon } from '@components/common';
 import { useAppTranslation } from '@hooks';
 
+const StyledArrowIcon = styled(Icon)<{ isExpanded: boolean }>`
+  width: 1.6rem;
+  fill: currentColor;
+  flex-shrink: 0;
+  transform: rotate(180deg);
+  transition: transform 200ms ease-in-out;
+
+  ${({ isExpanded }) =>
+    isExpanded &&
+    `
+    transform: rotate(0);
+  `}
```

Comment:
nit: Is this how lint/prettier prefers it? Looks quite hard to read

Reply:
Yeah. 
I would love this
<img width="356" alt="image" src="https://user-images.githubusercontent.com/78362532/164891497-ca12602b-0fe4-487c-8168-1456303d6f83.png">

But prettier gets really angry at that.
Maybe @FiboApe can create/edit a rule for this? He is really good with lint rules

Reply:
100%, that would be the way to format that in a much more readable way